.github/workflows/send_pr_notification.yml

```diff
@@ -0,0 +1,19 @@
+name: Send notification on PR opened
+
+on:
+  pull_request_target:
+    types: [labeled]
+
+jobs:
+  send_notification:
+    if: ${{ github.event.label.name }} == "Status:\ Reviewer"
+    runs-on: ubuntu-latest
+    steps:
+      - name: notify
+        run: |
+          curl -X POST ${{ secrets.SLACK_PR_NOTIFICATION_WEBHOOK }} \
+               -H 'Content-Type: application/json' \
+               -d '{ "GITHUB_PR_URL": "${{ github.event.pull_request.html_url }}",
+                     "PR_ADDITIONS": "${{ github.event.pull_request.additions }}",
```

Comment:
what does this refer to? receive notif when new commits are added to a PR?

Reply:
These are the fields related to the PR's metrics that Slack requires in order to show the message filled in with number of deletions, additions and changed files. For example, `PR_ADDITIONS` is the name of the field that Slack expects to find in the body of the POST in order to be able to send number of additions in the message we receive in Slack.

Reply:
The name of these fields can be configured in the Slack workflow. Also the number of fields that should exist in the body of the POST is configurable. This is just my choice for getting this working according to current requirements.

Reply:
ok, gotcha! now it is clear
thanks for the commit message update